gosling/api.py

```diff
@@ -109,12 +163,19 @@ def stack(*tracks, **kwargs):
     return View(alignment="stack", tracks=tracks, **kwargs)
 
 
-class Track(_EncodingMixin, _PropertiesMixen, mixins.MarkMethodMixin, core.Track):
+class Track(
+    _EncodingMixin,
+    _PropertiesMixen,
+    _TransformsMixen,
```

Comment:
I think we can instead force users to specify transforms in the track level only.

Reply:
It's a good question. Want to open an issue to explore this? In the case of `SingleTrack`, type is more narrow (than `Track`), but the other union types inherit from `Partial<SingleTrack>`. This means that we can force the strictness (completeness in this case) in our `Track` type by making it inherit from `SingleTrack`. However, it seems to me that `MultipleViews` is quite a different type compared to `SingleView`, so this could be challenging.

Reply:
I see. Allowing to support transforms in overlaid and stacked tracks in Gosling is mostly for avoiding code duplications, so I think allowing to use transforms only in `Track()` in the package I think makes sense.